packages/fxa-auth-server/package.json

```diff
@@ -92,6 +91,8 @@
     "hot-shots": "^9.0.0",
     "i18n-iso-countries": "^7.4.0",
     "ioredis": "^4.28.2",
+    "jed": "0.5.4",
```

Comment:
Mildly curious. What is [**jed**](https://www.npmjs.com/package/jed) package being used for? Only references I found while searching this page were this and 3 instances in yarn.lock file, so wasn't sure if this is actually being used in the project, or was an odd peer dependencies or ??